File: src/test/java/com/twcrone/spacemines/mine/MineFieldTests.java
package com.twcrone.spacemines.mine;

import org.junit.Test;

public class MineFieldTests {

	@Test
	public void createMineField() {

		MineFieldEntity mineField = new MineFieldEntity(3, 3, 3);

		assert mineField.getxLength() == 3;

	}

}

File: src/test/kotlin/com/twcrone/spacemines/mine/MineFieldTests.kt
package com.twcrone.spacemines.mine;

import org.junit.Test
import org.junit.Assert.*

class MineFieldTests {

	@Test
	fun `create a minefield with dimensions`() {

		val mineField = MineField(3, 3, 3)

		assertEquals(3, mineField.getxLength())
	}

}
